pkg/api/componentreadiness/component_report.go

```diff
@@ -1430,17 +1435,56 @@ func (c *componentReportGenerator) generateComponentTestReport(ctx context.Conte
 		}
 		updateCellStatus(rowIdentifications, columnIdentifications, testID, testStats, aggregatedStatus, allRows, allColumns, triagedIncidents, c.openRegressions)
 	}
-	// Those sample ones are missing base stats
+
+	// Anything we saw in the basis was removed above, all that remains are tests with no basis, typically new
+	// tests, or tests that were renamed without submitting a rename to the test mapping repo.
 	for testIdentification, sampleStats := range sampleStatus {
 		testID, err := buildTestID(sampleStats, testIdentification)
 		if err != nil {
 			return crtype.ComponentReport{}, err
 		}
+
+		// Check for approved regressions, and triaged incidents, which may adjust our counts and pass rate:
+		var testStats crtype.ReportTestStats
+		var triagedIncidents []crtype.TriagedIncident
+		var resolvedIssueCompensation int // triaged job run failures to ignore
+		// look for corresponding regressions we can account for in the analysis
+		approvedRegression := regressionallowances.IntentionalRegressionFor(c.SampleRelease.Release, testID.ColumnIdentification, testID.TestID)
+		baseRegression := regressionallowances.IntentionalRegressionFor(c.BaseRelease.Release, testID.ColumnIdentification, testID.TestID)
```

Comment:
Do you need this?  We don't have basis in this context and you pass in 0's below.  Should we remove this and just pass nil?

Reply:
Makes sense, changing to nil.